src/main/scala/com/blackmorse/spark/clickhouse/package.scala

```diff
@@ -8,8 +8,11 @@ package object clickhouse {
   val TABLE = "TABLE"
   val BATCH_SIZE = "BATCH_SIZE"
   val CLUSTER = "CLUSTER"
+  val READ_DIRECTLY_FROM_DISTRIBUTED_TABLE = "read_directly_from_distributed"
   val DIRECTLY_USE_DISTRIBUTED_TABLE = "read_directly_from_distributed"
   val RANDOM_WRITES_SHUFFLE = "random_writes_shuffle"
+  val USE_FORCE_COLLAPSING_MODIFIER = "use_force_collapsing_modifier"
```

Comment:
I believe `FORCE_COLLAPSING_MODIFIER` is enough.

Reply:
Done